test/array_util_test.rb

```diff
@@ -0,0 +1,76 @@
+require_relative "../src/array_util"
+require "test/unit"
+
+class TestArrayUtil < Test::Unit::TestCase
+
+# Test sum_even_numbers
+
+  def test_sum_even_numbers_nil_value_returns_nil
+    assert_nil(ArrayUtil.sum_even_numbers(nil))
```

Comment:
Superfluous test, the problem statement clearly states that you are given an array, so you can safely expect an array.

Now that you have it you need to improve the implementation, see my comment.